servatrice/src/serversocketinterface.cpp

```diff
@@ -877,63 +877,77 @@ Response::ResponseCode AbstractServerSocketInterface::cmdRegisterAccount(const C
     QString clientId = QString::fromStdString(cmd.clientid());
     qDebug() << "Got register command: " << userName;
 
+    Response_Register *re = new Response_Register;
+    re->set_custom_reason_str(servatrice->getCustomRegErrorMessage().toStdString());
+
     bool registrationEnabled = settingsCache->value("registration/enabled", false).toBool();
-    if (!registrationEnabled)
+    if (!registrationEnabled) {
+        rc.setResponseExtension(re);
```

Comment:
Branches like this simply returned a response value but didnt define any extended pb response items.  If the server operator has a custom string in the ini we now have to set the response extension results before returning the response and thus adding an additional line to the if statement requiring brackets around the beg/end of the logic tree.